ffi/Cargo.toml

```diff
@@ -15,6 +15,7 @@ crate-type = ["cdylib"]
 [features]
 default = []
 tsssp = ["sspi/tsssp"]
+scard = ["sspi/scard"]
```

Comment:
I’m not sure if this should be feature-gated for the DLL. Don’t we want to package the smart card login support for all Windows builds?
Any input @awakecoding

Reply:
After discussing with Marc-André, we agreed to keep this behind a feature gate. The main motivation is to not include the new smart card code into production build while it’s not ready yet.